channels/drdynvc/tsmf/ffmpeg/tsmf_ffmpeg.c

```diff
@@ -35,12 +35,22 @@
 #define AVMEDIA_TYPE_AUDIO 1
 #endif
 
+#if LIBAVCODEC_VERSION_MAJOR < 54
+#define MAX_AUDIO_FRAME_SIZE AVCODEC_MAX_AUDIO_FRAME_SIZE
+#else
+#define MAX_AUDIO_FRAME_SIZE 192000
+#endif
+
 typedef struct _TSMFFFmpegDecoder
 {
 	ITSMFDecoder iface;
 
 	int media_type;
-	enum CodecID codec_id;
+#if LIBAVCODEC_VERSION_MAJOR < 55
```

Comment:
I prefer not to pollute the code with inline `ifdef`s 
we should use the latest API and backport it using our own defines (at the head of this file or smthg).

For example 
```c
#if LIBAVCODEC_VERSION_MAJOR < 55
#define AVCodecID CodecID
#endif
```